python/lsst/ts/watcher/alarm.py

```diff
@@ -223,6 +230,42 @@ def close(self):
         self._cancel_escalation_timer()
         self._cancel_unmute()
 
+    async def make_log_entry(self, log_server_url):
+        """Post message to narrative log entry in response to alarm"""
+        now = datetime.datetime.now().isoformat()  # use ts_utils repo?
+        # Required? fields in payload:
+        #   message_text, level, user_id, user_agent, is_human
+        msg = f"alarm:{self.name} severity={self.severity}"
+        user = "make_log_entry"
+        payload = {
+            "message_text": msg,
+            "level": 0,
+            "user_id": "admin@love1.tu.lsst.org",  # TODO
+            "user_agent": user,
+            "is_human": False,
+            "tags": ["watcher", "alarm", "make_log_entry"],
+            "date_begin": now,
+            "date_end": now,
+            # ## Let following fields be defaulted.
+            # 'systems': ['ObservatorySoftwareTools'],
+            # 'subsystems': ['LOVE'],  # TODO
+            # 'cscs': [],
+            # 'urls': [],
+            # 'time_lost': 0,
+        }
+
+        url = f"{log_server_url}/messages"
+        print(f"Posting to log: {url=} {payload=}")  # TODO REMOVE!!!
+
+        # AIOHTTP docs say don't create session per request. We do so anyhow.
+        # Use default timeout (=5 minutes for whole operation)
```

Comment:
I don't see any timeout below. Also, 5 minutes is a lot! I would expect at most 10-15s.

Reply:
The comment means that by not specifying a timeout, I accepted the default which is 5 minutes according to the doc.  This is for whole operation (connect, write, response).

Reply:
Maybe expand your comment to clarify this point then? it is not clear from the text above. I would suggest something like:

```
        # AIOHTTP docs say don't create session per request. We do so anyhow.
        # By not specifying a timeout, we accept the default value of 5 minutes (according to the doc)
        # for the whole operation (connect, write, response).
```